cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/SystemArguments.java

```diff
@@ -100,6 +100,9 @@ public final class SystemArguments {
 
   public static final String NAMESPACE_CONFIG_PREFIX = "system.namespace.config.";
 
+  // Runtime argument to skip cleanup after pipeline run. Applicable only to jobs running natively in kubernetes.
+  public static final String K8S_JOB_SKIP_CLEANUP = "system.job.skipCleanup";
```

Comment:
The naming of the variable and the constant should match. See others as example. Also, the variable name suggested that it is only for k8s, but the value seems like it's generic?

Reply:
Updated the var name to SYSTEM_JOB_SKIP_CLEANUP. I meant to keep this runtime argument generic for future reuse.